lib/scripts/1166_fix_page_count.rb

```diff
@@ -0,0 +1,29 @@
+# Fix page_count for records from provided csv list.
+# From the root of project, run:
+#   `bundle exec rails r lib/scripts/1166_fix_page_count.rb`
+
+require 'csv'
+
+data = CSV.read('./filepaths_of_records_w_invalid_page_count.csv', headers: true)
+
+updated = 0
+data.each do |row|
+  begin
+    generic_file = GenericFile.find(row["dh_id"])
+  rescue ActiveFedora::ObjectNotFoundError
+    puts "Could not find record for #{row["dh_id"]}"
+    next
+  end
+
+  begin
+    generic_file.page_count = [`pdfinfo #{row["filepath"]} | grep "Pages" | cut -d":" -f 2 | sed 's/ //g'`]
```

Comment:
`pdfinfo` is missing from the production and staging servers. Oddly, I found an old script that uses `pdfinfo` the same way, but must have been removed at some point. IT ticket to install package is opened. Needs a quick test by hand when `pdfinfo` is available.